tests/translator/input/error_http_api_invalid_iam_auth.yaml

```diff
@@ -0,0 +1,37 @@
+Resources:
```

Comment:
I know you added integration tests (which thank you!) but could we add some of these input/output tests for valid cases? It is much easier to run these tests as quick validations of the translations.

Reply:
I'll look into that sure